packages/apm/src/integrations/tracing.ts

```diff
@@ -345,7 +323,7 @@ export class Tracing implements Integration {
          * If an error or unhandled promise occurs, we mark the active transaction as failed
          */
         logger.log(`[Tracing] Global error occured, setting status in transaction: ${SpanStatus.InternalError}`);
-        (Tracing._activeTransaction as SpanClass).setStatus(SpanStatus.InternalError);
```

Comment:
When pairing we removed the renaming in the import:

```diff
- import { Span as SpanClass } from '../span';
+ import { Span } from '../span';
```

And that change requires removing other occurrences of `SpanClass` in this file, but I can't find those changes.

Reply:
We need it in a few places still